.travis.yml

```diff
@@ -9,24 +7,35 @@ php:
   - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 env:
   - TRANSPORT=jackrabbit PACKAGE_VERSION=high
   - TRANSPORT=doctrine_dbal PACKAGE_VERSION=high
 
 matrix:
-  allow_failures:
-    - php: 7.0
   include:
     - php: 5.3
       env: TRANSPORT=doctrine_dbal COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.3
       env: TRANSPORT=jackrabbit COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.6
-      env: TRANSPORT=doctrine_dbal SYMFONY="3"
+    - php: 7.0
+      env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="2.7"
+    - php: 7.0
+      env: TRANSPORT=jackrabbit SYMFONY_VERSION="2.7"
+    - php: 7.0
+      env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="2.8"
+    - php: 7.0
+      env: TRANSPORT=jackrabbit SYMFONY_VERSION="2.8"
```

Comment:
do you mean to remove it to reduce the number of builds? would it make sense to do symfony 2.7 with dbal and 2.8 with jackrabbit?